packages/crypto/src/validation/validator.ts

```diff
@@ -76,13 +76,20 @@ export class Validator {
         this.rules[name] = implementation;
     }
 
+    /**
+     * Reset any previous results.
+     */
+    public __reset() {
```

Comment:
Public methods have no prefixes, if you read the issue.

Reply:
branch updated